extract_verbs.sh

```diff
@@ -16,6 +16,7 @@ fi
 LANGUAGE="$(echo $1 | tr '[:upper:]' '[:lower:]')"
 
 # Switch statement to select language
+verbTag='[[:space:]]VER[[:space:]]'
```

Comment:
The regex is not correct, there are no spaces around verb tags. Please remove `[[:space:]]`